src/FieldTrait.php

```
@@ -174,37 +172,34 @@ public function fieldFillWysiwyg(string $field, string $value): void {
 
     $element_id = $element->getAttribute('id');
     if (empty($element_id)) {
-      throw new ElementHtmlException('ID is empty', $driver, $element);
+      throw new \Exception('WYSIWYG field must have an ID attribute.');
     }
 
     $parent_element = $element->getParent();
```

PHPMD (2.15.0)

The variable $parent_element is not named in camelCase. (undefined)

(CamelCaseVariableName)